pkg/models/server_group.go

```diff
@@ -321,7 +321,7 @@ func (self *ServerGroup) AddServer(coordConn zkhelper.Conn, s *Server) error {
 		}
 	} else if s.Type == SERVER_TYPE_SLAVE && len(masterAddr) > 0 {
 		// send command slaveof to slave
-		err := utils.SlaveOf(s.Addr, masterAddr)
+		err := utils.SlaveOf(s.Addr, masterAddr, password, masterPassword)
```

Comment:
yeah， and i think that we have two passwords, proxypassword and serverpassword, shoule me remove proxypassword, only use serverpassword for qdb/codis-server?